api/logic/tag_janitor_test.go

```diff
@@ -39,33 +48,17 @@ func TestTagJanitorPulse(t *testing.T) {
 
 	//expecting 3 tags for task1 to be deleted from the tagstore
 	testutils.AssertEqual(t, tagsAdded-3, len(tags))
-	testutils.AssertEqual(t, "task2", tags[0].EntityID)
+	testutils.AssertEqual(t, "t2", tags[0].EntityID)
 }
 
-func getTagStore() (tag_store.TagStore, int) {
+func getTagStore(tags []models.Tag) (tag_store.TagStore, int) {
```

Comment:
yea I agree - it is redundant. i thought having an explicit variable named tagsAdded in makes the intent clearer. I don't feel strongly about it though. remove?
```
testutils.AssertEqual(t, tagsAdded-3, len(tags))
vs
testutils.AssertEqual(t, len(taskTags)-3, len(tags))
```